web/templates/account/edit.html.eex

```diff
@@ -0,0 +1,15 @@
+<h1>Edit User</h1>
+
+<%= render Notox.UserView, "form.html",
+  conn: @conn,
+  changeset: @changeset,
+  action: account_path(@conn, :update, @user),
```

Comment:
Passing in the user here is superfluous (you don't use the `id` to find the user in the controller). Note how this gets cleaned up once you switch to singleton resources.